2-mandatory/3-financial-times.js

```diff
@@ -14,24 +27,52 @@ function potentialHeadlines(allArticleTitles) {
     (you can assume words will always be seperated by a space)
 */
 function titleWithFewestWords(allArticleTitles) {
-    // TODO
+  // TODO
+  let fewestWordArticle = allArticleTitles[0];
+  for (let i = 0; i < allArticleTitles.length; i++) {
+    if (allArticleTitles[i].length < fewestWordArticle.length) {
+      fewestWordArticle = allArticleTitles[i];
```

Comment:
this is so close! this is a really good solution for if we are checking for the fewest letters, but here we are checking for the fewest _words_. Using .split(" ") here would help with this, for example:
` if (allArticleTitles[i].split(" ").length < fewestWordArticle.split(" ").length) {`

Reply:
thank  you so much. I got the point.